### ISSUE TITLE
Compatibility with node 0.10x is broken in v6.15.4

### ISSUE THREAD

**Issue by InvictusMB** [coderaiser/cloudcmd#121]
* **Version** (`cloudcmd -v`): 6.15.4
* **Node Version** `node -v`: 0.12
* **OS** (`uname -a` on Linux): Mac OS

```
/node_modules/currify/lib/currify.js:3
const f = (fn) => [
^^^^^
SyntaxError: Use of const in strict mode.
    at exports.runInThisContext (vm.js:73:16)
    at Module._compile (module.js:443:25)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/node_modules/cloudcmd/server_/cloudcmd.js:17:15)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/node_modules/cloudcmd/legacy/server/cloudcmd.js:1:80)
    at Module._compile (module.js:460:26)
```

I believe the offending line is [this](https://github.com/coderaiser/cloudcmd/blame/v6.15.4/server/cloudcmd.js#L17) as cloudcmd@6 code should require `currify/legacy`

**Comment by coderaiser** [comment]
Thank you, fixed in [currify v2.0.5](https://github.com/coderaiser/currify/releases/tag/v2.0.5).
Could you tell me please why are you using `node v0.12`?

**Comment by coderaiser** [comment]
Fixed in [v6.15.5](https://github.com/coderaiser/cloudcmd/releases/tag/v6.15.5).

**Comment by InvictusMB** [comment]
Legacy embedded system with complex certification procedure maintained by 3rd party vendor that loads nodejs application as a plugin.
cloudcmd is actually used to do some reverse engineering on that system by exposing fs as said vendor no longer knows how his stuff works.
